README_LINUX.md

```diff
@@ -206,145 +331,138 @@ not in the `lib` directory but in a suffixed one, e.g. `lib64`.
 ```shell
-$> export QT_QPA_PLATFORM=offscreen
-$> sclang
+export QT_QPA_PLATFORM=offscreen
+sclang
 ```
 
-Environment
------------
+On platforms without any display server at all it may be necessary to build SuperCollider without `X11` and `Qt` respectively - refer to the build flags to disable those features while configuring the build.
+
+## Environment variables
 
 The jack audio driver interface is configured based on various
 environment variables:
 
- * SC_JACK_DEFAULT_INPUTS comma-separated list of jack ports that the server's inputs should connect by default
-
-   ```
-   $> export SC_JACK_DEFAULT_INPUTS="system:capture_1,system:capture_2"
-   ```
-
-   in order to connect the first ports of one jack client, it is possible to specify only the client name
+- `SC_JACK_DEFAULT_INPUTS` comma-separated list of jack ports that the server's inputs should connect by default
 
-   ```
-   $> export SC_JACK_DEFAULT_INPUTS="system"
-   ```
+  ```shell
+  export SC_JACK_DEFAULT_INPUTS="system:capture_1,system:capture_2"
+  ```
 
- * SC_JACK_DEFAULT_OUTPUTS comma-separated list of jack ports that the server's outputs should be connected to by default.
+  in order to connect the first ports of one jack client, it is possible to specify only the client name
 
-   ```
-   $> export SC_JACK_DEFAULT_OUTPUTS="system:playback_1,system:playback_2"
-   ```
+  ```shell
+  export SC_JACK_DEFAULT_INPUTS="system"
+  ```
 
-   In order to connect the first ports of one jack client, it is possible to specify only the client name
+- `SC_JACK_DEFAULT_OUTPUTS` comma-separated list of jack ports that the server's outputs should be connected to by default.
 
-   ```
-   $> export SC_JACK_DEFAULT_OUTPUTS="system"
-   ```
+  ```shell
+  export SC_JACK_DEFAULT_OUTPUTS="system:playback_1,system:playback_2"
+  ```
 
-Two additional environment variables substitute directories for the default
-search path for plugins and synth definitions, respectively. Directory
-names are separated by ':' as in the Unix PATH variable:
+  In order to connect the first ports of one jack client, it is possible to specify only the client name
 
- * SC_PLUGIN_PATH, SC_SYNTHDEF_PATH
+  ```shell
+  export SC_JACK_DEFAULT_OUTPUTS="system"
+  ```
 
-   ```
-   $> export SC_SYNTHDEF_PATH="./synthdefs:/home/sk/SuperCollider/synthdefs"
-   ```
+- Two additional environment variables substitute directories for the default search path for plugins and synth definitions, respectively.
+  Directory names are separated by ':' as in the Unix PATH variable: `SC_PLUGIN_PATH`, `SC_SYNTHDEF_PATH`
 
+  ```shell
+  export SC_SYNTHDEF_PATH="./synthdefs:/home/sk/SuperCollider/synthdefs"
+  ```
 
-Contributors to this document
------------------------------
+## Contributors to this document
```

Comment:
How about - just a proposition

> Contributors to the original version of this document

To indicate this is not a current/exhaustive list...
Alternatively, add your names here?